test/edu-benefits/5490/schema/minimal-test.json

```diff
@@ -0,0 +1,89 @@
+{
+  "privacyAgreementAccepted": true,
+  "applicantInformation": {
+    "data": {
+      "relativeFullName": {
+        "first": "Jane",
+        "last": "Doe"
+      }
+    }
+  },
+  "benefitSelection": {
+    "data": {}
+  },
+  "previousBenefits": {
+    "data": {
+      "previousBenefits": {
+        "veteranFullName": {}
+      }
+    }
+  },
+  "sponsorVeteran": {
+    "data": {
+      "relativeFullName": {
```

Comment:
Ah, I think you're right, @raquelromano-gov. It's currently `relativeFullName` in the config right now, so I'll fix that up when I'm making the tweaks to the 5490.